src/en/components/file-uploader/use-case.md

```diff
@@ -10,11 +10,24 @@ eleventyNavigation:
   description: A file selector for document transfer.
   thumbnail: /images/en/components/preview/preview-file-uploader.svg
   alt: File uploader component shows grey boxes indicating text with an upload a file button, below is an uploaded file and a red remove button to remove the uploaded file.
```

Comment:
In the alt text for the preview image, we will need to change "upload a file button" to "choose file button".